OPERATING.md

```diff
@@ -87,6 +87,24 @@ Example:
         "_class" : "uk.nhs.digital.nhsconnect.lab.results.mesh.scheduler.SchedulerTimestamp"
     }
 
+## AWS DocumentDB TLS configuration
+
+AWS DocumentDB uses a private CA certificate and therefore requires a custom keystore to manage the CA certificates effectively.
+
+To use TLS, it has to be enabled in the DocumentDB instance with the Mongo connection string containing the `tls=true`. 
+For more information on TLS configuration in MongoDB see [TLS options for Mongo connection string](https://docs.mongodb.com/manual/reference/connection-string/#tls-options).
+
+SSH tunneling might require adding the connection string option: `tlsAllowInvalidHostnames=true` (as CA would try to resolve localhost as hostname).
+This option should only be used for local tests as this might create a vulnerability.
+
+Use of DocumentDB TLS requires java trust store to be provided manually. Instructions on how to create trust store can be found here: 
+[Connect to Document DB programmatically](https://docs.aws.amazon.com/documentdb/latest/developerguide/connect_programmatically.html#connect_programmatically-tls_enabled) 
+
+To configure custom trust store with AWS CA certificates set the `LAB_RESULTS_SSL_TRUST_STORE_URL` 
+with a runtime-accessible S3 URL with the JKS file (Example: s3://my-bucket/my-trust-store.jks). 
+The AWS resource running the adaptor must have sufficient permission to read this s3 object.
+Additionally, set the `LAB_RESULTS_SSL_TRUST_STORE_PASSWORD` to trust store password
```

Comment:
I would say either "set `variable`" or "set the `variable` environment variable"
```suggestion
To configure custom trust store with AWS CA certificates set `LAB_RESULTS_SSL_TRUST_STORE_URL` 
with a runtime-accessible S3 URL with the JKS file (Example: s3://my-bucket/my-trust-store.jks). 
The AWS resource running the adaptor must have sufficient permission to read this s3 object.
Additionally, set `LAB_RESULTS_SSL_TRUST_STORE_PASSWORD` to trust store password
```

```suggestion
To configure custom trust store with AWS CA certificates set the `LAB_RESULTS_SSL_TRUST_STORE_URL`
environment variable with a runtime-accessible S3 URL with the JKS file
(Example: s3://my-bucket/my-trust-store.jks). 
The AWS resource running the adaptor must have sufficient permission to read this s3 object.
Additionally, set the `LAB_RESULTS_SSL_TRUST_STORE_PASSWORD` environment variable to trust store password
```